### PR TITLE
style(content): #396 Fix spacing, colors, fonts

### LINKED ISSUE #396: Margins/fonts/color corrects in the UI
@bryanbell had a few nits about spacing and font sizes in the Recent Activity section and UI in general.

Bryan and @k88hudson to sit down in SF during work week to get this sorted and pair program the hell out of this thing.

### LINKED ISSUE #489: Fallback site icons for Highlights aren't square
URL in question is http://panamapaperscandal.com/david-cameron-shouldnt-resign-he-should-be-jailed/

[Embed.ly](http://embed.ly/docs/explore/extract?url=http%3A%2F%2Fpanamapaperscandal.com%2Fdavid-cameron-shouldnt-resign-he-should-be-jailed%2F) reports the `favicon_url` as `null`.

menubar

### LINKED ISSUE #498: Highlights boxes are different widths
The first column is wider than the other two:

screen shot 2016-04-08 composite

### LINKED ISSUE #502: Add better multi-line ellipsis based on character count  (Bug 1262435)
Bug 1262435 - Long titles are incorrectly truncated in "Highlights" section from "Timeline" and "New Tab"
via https://bugzilla.mozilla.org/show_bug.cgi?id=1262435

---

Created attachment 8738545 [details]
screenshot of the issue.PNG
image
#### [Affected versions]:
- Firefox 45.0 and up
- Activity Stream 1.0.7
#### [Affected Platforms]:
- Windows XP
#### [Prerequisites]:
- It will be best to use a clean profile.
- Latest Activity Stream add-on version (1.0.7) was previously installed on the used profile.
#### [Steps to reproduce]:
1. Start Firefox browser with the profile where Activity Stream add-on was previously installed.
2. Open a new tab and visit a page that has a long title (ex: http://goo.gl/EpguE8). 
3. Click on "Activity Stream" button from browser toolbar (if the website is not displayed in the Highlights from first try please refresh the page). 
4. Observe the title of the page in the "Highlights" area.
#### [Expected results]:

The title is correctly truncated when it exceeds the designed area.
#### [Actual results]:

The title is incorrectly truncated when it exceeds the designed area.
#### [Notes]:
- This issue is reproducible also in New Tab page.
- I've attached a screenshot of the issue.

### PR DESCRIPTION
**Pull request by k88hudson**
Fixes #396, Fixes #489, Fixes #498, Fixes #502 

This should also fix:

 #498 Highlight boxes are different sizes
 #502 Long titles have a bit of overflow on highlights
 #489 Fallback icons aren't square

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/activity-streams/511)

### CODE REVIEW

File: content-src/static/img/glyph-forward-16.svg [hunk 1]
```diff
@@ -7,7 +7,7 @@
       }
     </style>
   </defs>
-  <g id="glyph-forward-16">
+  <g id="glyph-forward-16-white">
```
Comment (pdehaan) [inline content-src/static/img/glyph-forward-16.svg]: Did we want to give this the same `id` as content-src/static/img/glyph-forward-16-white.svg?

Comment (k88hudson) [inline content-src/static/img/glyph-forward-16.svg]: oops!

**Comment by pdehaan** [comment]
LGTM! :+1:
Plus, favicons are finally visible for GitHub!

new_tab

**Comment by rlr** [comment]
woot :fireworks:

**Pull request merged**